data/notices.json

```diff
@@ -1,5 +1,17 @@
 {
   "notices": [
+    {
+      "title": "aws-iam: IAM OIDC provider used with EKS retrieves expiring certificate",
+      "issueNumber": 8607,
+      "overview": "An issue in the `aws-iam` module's `OpenIdConnectProvider` may break your application by the end of the year. Please see the issue thread for a workaround if you cannot upgrade when the fix is released.",
+      "components": [
+        {
+          "name": "framework",
```

Comment:
This can be a specific construct name. like `iam.OpenIdConnectProvider` so it only reaches the impacted users.

Reply:
Ahh, I see, thank you for letting me know.